docs/en/options.md

```diff
@@ -124,7 +124,7 @@ module.exports = {
 ### esModule
 
 - type: `boolean`
-- default: `false`
+- default: `true`
```

Comment:
`vue-loader` is released v12.x later.
We should be mentioned about the version like the below:

```
default: `false` (v13.0+, `true`)
```